index.html

```diff
@@ -49,17 +49,23 @@
 		</style>
 		<div class="navbar">
 			<ul>
-				<a href="#/app1">
+				<a onclick="routeToApp('/app1', event)" href="/app1">
```

Comment:
For this, we could use singleSpa.navigate, I think we could use [singleSpa.navigateToUrl](https://github.com/CanopyTax/single-spa/blob/master/docs/single-spa-api.md#navigatetourl) instead of implementing "routeToApp"

Reply:
Oh nice, didn't know that even existed.